.env.example

```diff
@@ -0,0 +1,3 @@
+NEXT_PUBLIC_GOOGLE_ANALYTICS=
```

Comment:
question: should we include any default values that have the structure of the string or comments telling people where they can get values for each field like in this pic

![Screen Shot 2022-03-27 at 5 01 24 PM](https://user-images.githubusercontent.com/33165426/160306903-9185386d-2351-48b1-9a5a-4ab0407b4c14.jpg)

Reply:
yes - i just made #228 to take care of this later